transport/kafka/kafka.go

```diff
@@ -54,6 +55,7 @@ func (d *KafkaDriver) Prepare() error {
 
 	//flag.StringVar(&d.kafkaKeying, "transport.kafka.key", "SamplerAddress,DstAS", "Kafka list of fields to do hashing on (partition) separated by commas")
 	flag.StringVar(&d.kafkaVersion, "transport.kafka.version", "2.8.0", "Kafka version")
+	flag.StringVar(&d.kafkaCompressionType, "transport.kafka.compression.type", "lz4", "Kafka default compression type")
```

Comment:
Could you leave it empty, seems that we are introducing a default.

Reply:
we can set the type=none by default to not enforce a new default parameter to the goflow2 producer

Reply:
change made